chef/data_bags/crowbar/migrate/provisioner/034_os_upgrading_state.rb

```diff
@@ -0,0 +1,11 @@
+def upgrade(ta, td, a, d)
```

Comment:
Unused method argument - `td`. If it's necessary, use `_` or `_td` as an argument name to indicate that it won't be used.